.github/workflows/spotbugs.yml

```diff
@@ -39,4 +39,4 @@ jobs:
 #          path: '**/spotbugsXml.xml'
 #      # Can be simplified post this issue is fixed https://github.com/jwgmeligmeyling/spotbugs-github-action/issues/9
       - name: Build with Maven for spotbugs check to mark build as fail if voilations found
-        run: mvn -Pbuild-with-spotbugs -B install --file pom.xml -DskipTests -Dmaven.javadoc.skip=true -Dspotbugs.failOnError=true
\ No newline at end of file
+        run: mvn -Pbuild-with-spotbugs -B install --file pom.xml -DskipTests -Dmaven.javadoc.skip=true -Dspotbugs.failOnError=true -pl '!software.amazon.lambda.examples:powertools-examples-idempotency,!software.amazon.lambda.examples:powertools-examples-batch,!software.amazon.lambda.examples:powertools-examples-cloudformation,!software.amazon.lambda.examples:powertools-examples-core-utilities-cdk,!software.amazon.lambda.examples:powertools-examples-core-utilities-sam,!software.amazon.lambda.examples:powertools-examples-core-utilities-serverless,!software.amazon.lambda.examples:powertools-examples-core-utilities-terraform,!software.amazon.lambda.examples:powertools-examples-parameters,!software.amazon.lambda.examples:powertools-examples-serialization,!software.amazon.lambda.examples:powertools-examples-validation,!software.amazon.lambda.examples:cdk,!software.amazon.lambda:powertools-examples'
```

Comment:
This one's a bit awkward - 1/ we still want to run spotbugs on everything and it is nice if the lowest common denominator JDK is used but 2/ the LCD-JDK is different for the modules themselves and the examples.

IMHO we should shift this up to `java-version: 11` on line 31, and remove this exclude. The JDK itself shouldn't really make a difference for the spotbugs output, and that way we are still covering the examples. What do you reckon @jdoherty ?